Exiled.API/Features/Round.cs

```diff
@@ -60,7 +60,12 @@ public static bool IsLobbyLocked
         /// </summary>
         public static void ForceEnd()
         {
-            if (IsStarted && Player.List.Count() > 1 && !IsLocked)
+            if (RoundSummary.singleton._keepRoundOnOne && Player.List.Count() < 2)
```

Comment:
```suggestion
            if (RoundSummary.singleton._keepRoundOnOne && Player.Dictionary.Count < 2)
```

Save our one CPU tick